### DISCUSSION TITLE
What we looking for here?

### DISCUSSION THREAD

**Discussion by ghost** [lmachens/soc.th.gl#29] [Ideas]
Hi. I'm two things. Developer (senior years) and admirer of SoC.

What kind of art direction are we looking for in the homesite? Maybe a more direct comparison between the pixel art in the game, with full-blown maps and some general information about each characters and its progression in the game. Or we could go with a more AoE style for the evolution of cities and units*. 

* = https://i.redd.it/vql38juhxch41.jpg

**Comment by lmachens** [comment]
Hi, I like to have a full codex of items, wielders, skills, ... including the lore of the game.
It should be easy to navigate and items should be linked between each other. Other than the in-game codex, we can display more detailed stats like item drop-rates, and we could add buildings and maps too.
This project targets to be a full wiki/codex, but have enough space for tools like skill-builder or item comparisons.
I am open for suggestions how to display this data. I prefer simplicity to complexity, especially for mobile users.

The home-page is discussed in #20.
